src/main/java/seedu/address/logic/commands/eventcommands/EventEditCommand.java

```diff
@@ -52,13 +50,12 @@ public class EventEditCommand extends Command {
             + "by the index number used in the displayed event list. "
             + "Existing values will be overwritten by the input values.\n"
             + "Parameters: INDEX (must be a positive integer) "
-            + "[" + PREFIX_NAME + "EVENT NAME] "
             + "[" + PREFIX_ROLE + "ROLES] "
             + "[" + PREFIX_START_DATETIME + "START DATE] "
             + "[" + PREFIX_LOCATION + "LOCATION]... "
             + "[" + PREFIX_MAX_VOLUNTEER_SIZE + "MAX VOLUNTEER COUNT]\n"
             + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_ROLE + "Director "
+            + PREFIX_ROLE + "1 Director "
```

Comment:
idk if can change this HAHA, isit feature flaw?

Reply:
i think its fine ba, after all our implementation where we have a number before the word is an old implementation

Reply:
ya sial, this one looks like a feature flaw although it is very minor

Reply:
it's wrong since a user using the old command format would get an error when running the example so fixing is fine haha